### DISCUSSION TITLE
Feed incorrectly reporting various PR activities as "labeled a pull request..."

### DISCUSSION THREAD

**Discussion by GoldenChrysus** [community/community#76760] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Couldn't get a full screenshot since there are literally 16 of these redundant items in my feed:
![opera_2023-11-23_195211866](https://github.com/community/community/assets/17366720/3205acbf-4220-4bc9-8c3d-86bac4c8db3b)
![opera_2023-11-23_200312028](https://github.com/community/community/assets/17366720/47b12065-16cc-417c-8527-8c295d85eb3f)

When I actually look at the PR, it looks like this was instead lots of various activity (comments, other labels added, etc.). Instead of showing me that information, you have decided to show me the same "labeled a pull request `good first issue`" message 16 times.

Either you need to fix the feed to show the actual activity occurring on the PRs, or if this is for some reason intentional, you probably need to rethink why you think showing someone the same message 16 times is beneficial to the user.

That's just on page 1. The feed is absolutely useless since I have to sift through duplicates. I clicked "More" and got 18 more of the same useless entries.

---

Also, even if the feed were correctly showing all of the activity, I probably don't need to see the entire PR preview every single time. e.g. on page 1, there are 6 entries for the same PR back to back; one page 2 there are 7 back to back. I don't need to see the PR preview 7 times. If back to back, these should be consolidated to show the 7 activities plus 1 instance of the PR preview.

**Comment by sphars** [comment]
I'm seeing this too. One of my starred repos opened a PR with the `good first issue` label, and now I'm seeing the same entry in my feed for how many comments that PR received. 

![image](https://github.com/community/community/assets/18315081/618da363-953a-47de-b3a6-a11237c572e8)

**Reply by Chaphasilor** [reply]
Same here. My feed is flooded with the same content over and over...

**Comment by mikerockett** [comment]
Seeing this as well, but feed is configured to only show releases.

<img width="927" alt="image" src="https://github.com/community/community/assets/4586280/ff45e3a3-bf85-4350-947f-547e6f968666">
